contracts/attesters/hydra-s1/variants/HydraS1AccountboundAttester.sol

```diff
@@ -1,24 +1,24 @@
 // SPDX-License-Identifier: MIT
```

Comment:
I think This file should be at the same level as the Simple attester because it's not a "variant"